src/stories/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import { ButtonProps } from "./interfaces";
+
+const PrimaryButton = (props: ButtonProps) => {
```

Comment:
If primary and secondary buttons are completely different, it makes sense to have different components `<PrimaryButton />`, `<SecondaryButton />` etc. Otherwise, let's call it just a `Button` and provide a prop for `type` - `"primary" | "secondary"`

Reply:
Changed <PrimaryButton /> to <Button />

I guess we can add additional type when we will need it.